spec/models/concerns/lead_school_migratable_spec.rb

```diff
@@ -3,55 +3,55 @@
 require "rails_helper"
 
 describe LeadSchoolMigratable do
```

Comment:
If this is no longer needed, how about removing any remaining includes and delete the spec and the concern instead?

Reply:
Is this still being used in the `Funding::TraineeSummaryRow`?

Reply:
Yeah this is still being used for now. Once the TraineeSummaryRow has been refactored, then the whole thing can be removed.